ddtrace/llmobs/_llmobs.py

```diff
@@ -774,6 +777,153 @@ def _tag_metrics(span: Span, metrics: Dict[str, Any]) -> None:
             return
         span.set_tag_str(METRICS, safe_json(metrics))
 
+    @classmethod
+    def submit_evaluation_for(
+        cls,
+        label: str,
+        metric_type: str,
+        value: Union[str, int, float],
+        span: Optional[dict] = None,
+        span_with_tag: Optional[Tuple[str, str]] = None,
+        tags: Optional[Dict[str, str]] = None,
+        ml_app: Optional[str] = None,
+        timestamp_ms: Optional[int] = None,
+        metadata: Optional[Dict[str, object]] = None,
+    ) -> None:
+        """
+        Submits a custom evaluation metric for a given span.
+
+        :param str label: The name of the evaluation metric.
+        :param str metric_type: The type of the evaluation metric. One of "categorical", "score".
+        :param value: The value of the evaluation metric.
+                      Must be a string (categorical), integer (score), or float (score).
+        :param dict span: A dictionary of shape {'span_id': str, 'trace_id': str} uniquely identifying
+                            the span associated with this evaluation.
+        :param tuple span_with_tag: A tuple of shape (tag_key, tag_value) uniquely identifying
+                            the span associated with this evaluation.
+        :param tags: A dictionary of string key-value pairs to tag the evaluation metric with.
+        :param str ml_app: The name of the ML application
+        :param int timestamp_ms: The timestamp in milliseconds when the evaluation metric result was generated.
+        :param dict metadata: A JSON serializable dictionary of key-value metadata pairs relevant to the
+                                evaluation metric.
+        """
+        if cls.enabled is False:
+            log.warning(
```

Comment:
```suggestion
            log.debug(
```

It's a valid use-case to disable llmobs in which case this log will be very noisy as a `warning`

Reply:
I also will change the `ml_app` warning to debug as well, in cases where customers use this method in-code but rely on environment variables to enable llm obs

Reply:
One second thought, wydt of:
- submit eval called with llmobs disabled - debug log and return 
- submit eval called with missing api key or ml app - warning log and return

Reply:
Nevermind
- for the api key, we'll log when the writer fails to send
- for ml app, this is already captured by checking if llmobs is enabled (ml app is always present if llm obs is enabled)